test/api_integration/config.js

```diff
@@ -22,6 +22,12 @@ export default async function ({ readConfigFile }) {
     servers: commonConfig.get('servers'),
     junit: {
       reportName: 'API Integration Tests'
-    }
+    },
+    kibanaServerArgs: [
```

Comment:
:+1:

Reply:
glad you like this! it was one of the changes i had a lot of ideas about.